content/blog/explore-the-hpe-greenlake-for-compute-ops-management-rest-api-using-python-and-powershell.md

```diff
@@ -0,0 +1,242 @@
+---
+title: Explore the HPE GreenLake for Compute Ops Management REST API using
+  Python and PowerShell
+date: 2022-12-01T12:28:58.682Z
+author: Vincent Berger & Frederic Passeron
+authorimage: /img/vb-fp192.png
+disable: false
+tags:
+  - "hpe-greenlake "
+  - "hpe-greenlake-for-compute-ops-management "
+  - "developer "
+  - devops
+---
+HPE GreenLake for Compute Ops Management automates and transforms complex and time-consuming compute management operations into a simplified experience across edge-to-cloud. Whether you are an IT OPS or a DEV OPS engineer, you know that automation is the key to success. And today’s automation relies heavily on APIs and how one can interact easily with them. So, let us show you how to leverage the API provided so you, too, can take advantage of what HPE GreenLake for Compute Ops Management can provide.
```

Comment:
Seeing as how this introduction is exactly the same as the one in your other blog post, perhaps you should add a line at the end of each of these that shows the difference between the two blog posts. For instance, for this one, consider adding in "In this blog post, we will cover how to do this using Python and PowerShell." and for the other, add in a line at the end of the first paragraph that says "In this blog post, we will cover how to do this using cURL and Postman."